dbActions.py

```diff
@@ -0,0 +1,71 @@
+import sqlite3
+import Settings
+
+
+def create_table(con):
+    """
+    create a table in case it is not exist
+    :param con
+    """
+    with con:
+        cur = con.cursor()
+        cur.execute("CREATE TABLE IF NOT EXISTS scores(Name TEXT, score INTEGER);")
+
+
+def insert_new_score(name, level, con, table_name):
+    """
+    by the user's score, will insert his name or his highest score
+    :param name
+    :param level
+    :param con
+    :param table_name
+    """
+    score = get_score(table_name, name, con)
+    if score == None:
```

Comment:
score is None.
Check out [this one](http://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is-in-python?lq=1) and [this one](http://stackoverflow.com/questions/3965104/not-none-test-in-python)